common/message.ts

```diff
@@ -24,26 +29,24 @@ export const enum TelemetryInfoType {
   VPD = 'vpd',
 }
 
-export const enum ResponseErrorInfoMessage {
+export const enum ResponseErrorInfoMessage { // not sure if each word should be connected with '-'
```

Comment:
I don't think the comment belongs here. Lets address this offline and delete the comment.

Reply:
Done.